parser/model/model.go

```diff
@@ -412,6 +451,7 @@ type TableInfo struct {
 	AutoRandID      int64  `json:"auto_rand_id"`
 	MaxColumnID     int64  `json:"max_col_id"`
 	MaxIndexID      int64  `json:"max_idx_id"`
+	MaxForeignKeyID int64  `json:"max_fk_id"`
```

Comment:
Could we use `MaxIndexID` instead of this?

Reply:
How about use `MaxConstraintID` and remove `MaxForeignKeyID `. Since `MaxConstraintID` already exist and never used. I want to make `MaxIndexID ` only used for the index.

Reply:
@tangenta PTAL

Reply:
I prefer to keep `MaxForeignKeyID`.

In MySQL, the constraints don't share the same ID:

```sql
mysql> create table t (a int, check(a != 1), foreign key (a) references t1(a));
Query OK, 0 rows affected (0.05 sec)

mysql> show create table t;
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table | Create Table                                                                                                                                                                                                                              |
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| t     | CREATE TABLE `t` (
  `a` int DEFAULT NULL,
  KEY `a` (`a`),
  CONSTRAINT `t_ibfk_1` FOREIGN KEY (`a`) REFERENCES `t1` (`a`),
  CONSTRAINT `t_chk_1` CHECK ((`a` <> 1))
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci |
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```